README.md

```diff
@@ -139,6 +142,89 @@ To upgrade this resource from `4.0.6` to `4.2.7`:
    kubectl apply -f <example>.yaml --namespace <my-namespace>
    ```
 
+## Secure MongoDB Resource Connections using TLS
+
+You can configure the MongoDB Community Kubernetes Operator to use TLS certificates to encrypt traffic between:
+
+- MongoDB hosts in a replica set, and
+- Client applications and MongoDB deployments.
+
+### Prerequisites
+
+Before you secure MongoDB resource connections using TLS, you must:
+
+1. Using your own Certificate Authority (CA), create a PEM-encoded TLS certificate for each member of the Application Database’s replica set. The certificate must have one of the following:
```

Comment:
There is only a single certificate and private key used for all members. The user only needs to create a single one and not one for each member.

Reply:
I think I addressed this in https://github.com/mongodb/mongodb-kubernetes-operator/pull/128/files#diff-04c6e90faac2675aa89e2176d2eec7d8R156